### PR TITLE
Refactored event creation

### PR DESCRIPTION
**Pull request by dinodsaurus**
This one is connected to #9 , I will also add tests and timezone to the list output so dont merge yet :+1:

### CODE REVIEW

**Comment by martinheidegger** [comment]
The work here looks very good, looking forward to the end result. _Minor note: It would be awesome if the refactoring steps would be single commits to be easier to follow_

File: lib/chapter/event-create.js [hunk 1]
```diff
@@ -121,187 +43,22 @@ module.exports = {
...
-                  return 'The end time needs to be after the start time.'
-                } else {
-                  return true
-                }
-              }
-              return result
-            }
-          }
+          eventHelper.askForUrl,
+          eventHelper.askForName(input, suggestions, locations),
```
Comment (martinheidegger) [inline lib/chapter/event-create.js:36]: Couldn't it be even better if `askForName`(etc.) would not take `input` but rather the `input[...]` properties that they need?

Comment (dinodsaurus) [inline lib/chapter/event-create.js:36]: I think its overkill, but if you think its important i can rewrite it :+1:

**Comment by dinodsaurus** [comment]
~~hmmm, i think there is some issue with travis or something Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version GLIBCXX_3.4.20 not found~~

ok fixed it by adding some addons to our travis build

**Comment by martinheidegger** [comment]
Would you consider this now ready to merge?

**Comment by dinodsaurus** [comment]
yep, I think we can merge this one

**Comment by martinheidegger** [comment]
Tested locally, seems to work fine! Merged.

**Pull request merged**